tests/sources/test_spot_price_sources.py

```diff
@@ -293,3 +295,23 @@ async def test_failed_price_service_request():
 
     assert v is None
     assert t is None
+
+
+@pytest.mark.asyncio
+async def test_coinpaprika():
+    """Test Coinpaprika price service"""
+    # Example response from Coinpaprika API
+    # TODO: consider using a mock responses to lower api calls for any test not just this api
```

Comment:
Can probably remove this todo comment, I've made it an issue [here](https://github.com/tellor-io/telliot-feeds/issues/637)

Reply:
Awesome, will do. Thank you!